core/base_service_test.go

```diff
@@ -1003,6 +1003,87 @@ func TestExtConfigFromVCAP(t *testing.T) {
 	clearTestVCAP()
 }
 
+func TestConfigureServiceFromCredFile(t *testing.T) {
```

Comment:
Let's discuss why you needed to add a test that explicitly calls `ConfigureService` after calling `NewBaseService` since `NewBaseService` should call `ConfigureService`.

Reply:
oh wait, you are calling ConfigureService with a different service name... clever :)
you almost fooled me, but not quite... haha

Reply:
yeah my intention was to have this test there once we remove ConfigureService out of the constructor. Technically when we do that the test should still pass